config/environments/development.rb

```diff
@@ -37,6 +38,12 @@
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
 
+  config.action_mailer.delivery_method = :smtp
```

Comment:
I don't think we want to send emails in development mode.

Reply:
:+1:

Reply:
Removed.